static/bidder-info/connatix.yaml

```diff
@@ -0,0 +1,20 @@
+endpoint: "https://capi.connatix.com/rtb/ortb"
+maintainer:
+  email: "pubsolutions@connatix.com"
```

Comment:
send email for verification, please respond

Reply:
done